dali/kernels/signal/decibel/to_decibels_cpu.cc

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "dali/kernels/signal/decibel/to_decibels_cpu.h"
+#include <cmath>
+#include <complex>
+#include <vector>
+#include "dali/core/common.h"
+#include "dali/core/error_handling.h"
+#include "dali/core/format.h"
+#include "dali/kernels/kernel.h"
+#include "dali/kernels/signal/decibel/decibel_calculator.h"
+
+namespace dali {
+namespace kernels {
+namespace signal {
+
+template <typename T, int Dims>
+ToDecibelsCpu<T, Dims>::~ToDecibelsCpu() = default;
+
+template <typename T, int Dims>
+KernelRequirements ToDecibelsCpu<T, Dims>::Setup(
+    KernelContext &context,
+    const InTensorCPU<T, Dims> &in,
```

Comment:
shouldn't it be `const InTensorCPT<const T, Dims>`?

Reply:
No
```
template <typename StorageBackend, typename ElementType, int sample_dim>
using InTensor = TensorView<StorageBackend, const ElementType, sample_dim>;

template <typename ElementType, int sample_dim>
using InTensorCPU = InTensor<StorageCPU, ElementType, sample_dim>;
```